apps/backend/alembic/env.py

```
@@ -68,36 +69,45 @@ def include_name(name, type_, parent_names):
 
 
 def include_object(object, name, type_, reflected, compare_to):
     """
     Filter function for more detailed object inclusion control.
-    
+
     This ensures we only manage our application objects in migrations.
     """
     if type_ == "table" and reflected and compare_to is not None:
         # Skip TimescaleDB system tables
         if name.startswith(("_timescaledb", "timescaledb_")):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)